test/execute.js

```diff
@@ -529,13 +529,60 @@ describe('execute', () => {
         headers: {
           head: 'justTheHead',
         },
-        body: {
+        body: JSON.stringify({
           json: 'rulez'
-        }
+        })
       })
     })
   })
 
+  it('should use the whole parameters object as the parameter value if parameter is in the body', function () {
```

Comment:
@fehguy @webron anything new on this? The code for this feature was already there but was broken